tests/test_backend.py

```diff
@@ -301,6 +301,12 @@ def expect_fail(test_case_path):  # type: (str) -> None
 expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_floor_align_corners_cpu')
 expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu')
 
+# Pad errors - NGONNX-783
```

Comment:
You can remove the `NGONNX-783` annotation from here, as we will close this ticket.

Reply:
Done